live-examples/js-examples/regexp/regexp-assertions.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var text = "A quick fox";
+var regexpAhead = /[a-z]+ (?=fox)/;
+console.log(text.match(regexpAhead));
+// expected output [ 'quick ']
+
+var regexpNegativeAhead = /[A-Z]+ (?!slow)/
+console.log(text.match(regexpNegativeAhead));
+// expected output [ 'A ']
+
+var regexpBehind = /(?&lt;=quick) \w+/;
+console.log(text.match(regexpBehind));
+// expected output (only in Chrome) for now [ 'fox']
```

Comment:
I don't think we should include compat statements in the examples, they will get out of date. Just say "expected output" and hope people look at the compat data.